README.md

```diff
@@ -7,35 +7,35 @@ language that you can read [here](https://rust-unofficial.github.io/patterns/).
 
-* TODO iterators (to safely avoid bounds checks)
-* TODO closures and lifetimes (coupling to lifetime)
-* TODO platform-specific sub-modules (<https://github.com/rust-lang/rfcs/blob/master/text/0517-io-os-reform.md#platform-specific-opt-in>)
-* TODO Module organisation (by looking at examples such as Rusts `libstd`, and how it integrated into the Rusts source code, lessons can be learned about ergonomic project management and API design. Closely assosciated with platform-specific sub-modules)
-* [Entry API](patterns/entry.md) (Currently just a boilerplate)
-* TODO extension traits
-* TODO destructor bombs (ensure linear typing dynamically, e.g., <https://github.com/Munksgaard/session-types/commit/0f25ccb7c3bc9f65fa8eaf538233e8fe344a189a>)
-* TODO convertible to Foo trait for more generic generics (e.g., <http://static.rust-lang.org/doc/master/std/fs/struct.File.html#method.open>)
-* [Late bound bounds](patterns/late-bounds.md) (Currently just a boilerplate)
-* TODO 'shadow' borrowed version of struct - e.g., double buffering, Niko's parser generator
-* TODO composition of structs to please the borrow checker
-* TODO `Error` traits and `Result` forwarding
-* TODO graphs
+- TODO iterators (to safely avoid bounds checks)
+- TODO closures and lifetimes (coupling to lifetime)
+- TODO platform-specific sub-modules (<https://github.com/rust-lang/rfcs/blob/master/text/0517-io-os-reform.md#platform-specific-opt-in>)
+- TODO Module organisation (by looking at examples such as Rusts `libstd`, and how it integrated into the Rusts source code, lessons can be learned about ergonomic project management and API design. Closely assosciated with platform-specific sub-modules)
+- [Entry API](patterns/entry.md) (Currently just a boilerplate)
+- TODO extension traits
+- TODO destructor bombs (ensure linear typing dynamically, e.g., <https://github.com/Munksgaard/session-types/commit/0f25ccb7c3bc9f65fa8eaf538233e8fe344a189a>)
+- TODO convertible to Foo trait for more generic generics (e.g., <http://static.rust-lang.org/doc/master/std/fs/struct.File.html#method.open>)
+- [Late bound bounds](patterns/late-bounds.md) (Currently just a boilerplate)
+- TODO 'shadow' borrowed version of struct - e.g., double buffering, Niko's parser generator
+- TODO composition of structs to please the borrow checker
+- TODO `Error` traits and `Result` forwarding
+- TODO graphs
 
 ### Anti-patterns
 
-* TODO thread + catch_panic for exceptions
-* TODO Clone to satisfy the borrow checker
-* TODO Matching all fields of a struct (back compat)
-* TODO wildcard matches
-* TODO taking an enum rather than having multiple functions
-* TODO `unwrap()`ing every `Result` instead of forwarding it
+- TODO thread + catch_panic for exceptions
+- TODO Clone to satisfy the borrow checker
+- TODO Matching all fields of a struct (back compat)
+- TODO wildcard matches
+- TODO taking an enum rather than having multiple functions
+- TODO `unwrap()`ing every `Result` instead of forwarding it
```

Comment:
Can you remove that and link to the umbrella issue with a small sentence stating that people will find all the content to add there?

Reply:
I can do that in another PR and then we merge this one after, maybe easier and cleaner

Reply:
ok!